frontend/src/routes/SingleShowing/BoughtShowing.js

```diff
@@ -2,11 +2,12 @@ import React from "react";
 import Loader from "../../ProjectorLoader";
 import buildUserComponent from "./UserComponentBuilder";
 import MainButton from "../../MainButton";
+import ParticipantList from "./ParticipantsList";
 
 const UserItem = buildUserComponent(({ user }) =>
-  <div>
-    {user.nick || user.name} ({user.phone})
-  </div>
+  <span>
+    <b>{user.phone}</b> ({user.firstName} '{user.nick}' {user.lastName})
```

Comment:
Are we changing the format for usernames? Also, use a `<strong>`-tag instead of `<b>`